scripts/setup_macos.sh

```diff
@@ -192,16 +192,7 @@ if [[ "${_arg_modify_bashrc}" = on ]]; then
   echo -e "\n# Added by the garage installer" >> "${BASH_PROF}"
```

Comment:
please try this script on the cleanest mac you can find (or VM) and verify it works before submitting